pvlib/irradiance.py

```diff
@@ -2915,3 +2917,61 @@ def dni(ghi, dhi, zenith, clearsky_dni=None, clearsky_tolerance=1.1,
             (zenith < zenith_threshold_for_zero_dni) &
             (dni > max_dni)] = max_dni
     return dni
+
+
+def component_sum_irradiance(zenith,
+                             ghi=None,
+                             dhi=None,
+                             dni=None,
+                             clearsky_dni=None):
```

Comment:
Let's rename to `solar_zenith` and `dni_clear`, again for consistency with the rest of pvlib